### PR TITLE
Homework 4

### PR DESCRIPTION
**Pull request by kolosokvit**

### CODE REVIEW

File: src/main/java/ticket/Ticket.java [hunk 1]
```diff
@@ -33,11 +38,28 @@ public Ticket(String id, String concertHall, String eventCode, LocalDateTime tim
...
         this.time = time;
     }
 
+    public void share(String phone) {
+        System.out.println("Ticket is shared by phone: " + phone);
+    }
+
+    public void share(String phone, String email) {
+        System.out.println("Ticket is shared by phone: " + phone + " and email: " + email);
+    }
```
Comment (AtticSiol) [inline src/main/java/ticket/Ticket.java]: move it into the end

File: src/main/java/user/Client.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package user;
+
+import utils.ClassIdCounter;
+import interfaces.Printable;
+
+public class Client extends User implements Printable {
+    private static final int CLASS_ID = ClassIdCounter.getClassIdCounter();
```
Comment (AtticSiol) [inline src/main/java/user/Client.java]: the id should not be in the class level

File: src/main/java/ticket/Ticket.java (hunk 1 above)
Comment (kolosokvit) [reply src/main/java/ticket/Ticket.java]: Fixed in https://github.com/kolosokvit/ticket-service/pull/2/commits/9f202e8e46a97a90991e72e9e9a12acca3d7ae05

File: src/main/java/user/Client.java (hunk 2 above)
Comment (kolosokvit) [reply src/main/java/user/Client.java]: Fixed in https://github.com/kolosokvit/ticket-service/pull/2/commits/bb807d65db44ce15c9bf007a1e9cf711391956c4

**Review by AtticSiol — APPROVED** [review]

**Pull request merged**